src/cascadia/LocalTests_TerminalApp/SettingsTests.cpp

```diff
@@ -433,15 +433,17 @@ namespace TerminalAppLocalTests
             "globals": {
                 "alwaysShowTabs": true,
                 "initialCols" : 120,
-                "initialRows" : 30
+                "initialRows" : 30,
+                "rowsToScroll" :  4,
             }

```

Comment:
I'm not sure why this wasn't picked up by the build bot, but this test now fails for me when run locally because of the trailing comma (that's not valid JSON).